Makefile

```diff
@@ -37,6 +37,8 @@ endif
 
 SERVICES:=$(filter-out $(OPTIONS),$(ARGS))
 
+COMPOSE_DOWN:= docker-compose -p edgex -f docker-compose.yml -f docker-compose-no-secty-with-ui.yml down $1
```

Comment:
You will want to change the `:=` to just `=`. The `:=` will make it a pre-computed value and not a function.

Reply:
Nice to know. :-)